scripts/create-topics.sh

```diff
@@ -40,6 +40,8 @@ notification_topics=(
 for topic_name in "${notification_topics[@]}"; do
   create_topic "$topic_name" "${NOTIFICATION_TOPICS_PARTITIONS:-3}" "retention.ms=${NOTIFICATION_TOPICS_RETENTION_MS:-43200000}"
 done
+create_topic "${API_TOPIC_PREFIX:-}dtrack.notification.project-vuln-analysis-complete" "${NOTIFICATION_TOPICS_PARTITIONS:-3}" "cleanup.policy=compact" "retention.ms=${NOTIFICATION_TOPICS_RETENTION_MS:-43200000}"
```

Comment:
Have we also submitted the topic creation in MetaManager?

Reply:
Need to add same in TOPICS.md

Reply:
will also submit to metamanager